include/seqan3/search/algorithm/detail/search_scheme_algorithm.hpp

```diff
@@ -404,7 +404,7 @@ inline bool search_ss(iterator_t it, query_t & query,
         return true;
     }
     // Exact search in current block.
-    else if ((max_error_left_in_block == 0) && (rb - lb - 1 != blocks_length[block_id]) ||
+    else if ((max_error_left_in_block == 0 && (rb - lb - 1 != blocks_length[block_id])) ||
```

Comment:
Good idea to put brackets around the `&&` term but I would additionally keep the brackets around the first term, i.e.

```suggestion
    else if (((max_error_left_in_block == 0) && (rb - lb - 1 != blocks_length[block_id])) ||
```

Reply:
It was not a good idea from me, it was a compiler warning :laughing: . I don't see the increase in readability to put extra brackets, but will do!